app/views/shared/_navbar.html.slim

```diff
@@ -9,11 +9,11 @@ nav.navbar.navbar-expand-lg.navbar-dark.bg-dark
           li= link_to 'Български', url_for(locale: :bg), class:"nav-link", class:"nav-link"           
         - else 
           li= link_to 'English', url_for(locale: :en), class:"nav-link"
-        li= link_to "Car Diary", root_path, class:"nav-link"
+        li= link_to "Car Diary", root_path(locale: I18n.locale), class:"nav-link"
```

Comment:
hmm, I don't think it's a good idea to set a locale param for all urls

Reply:
How we should fix that problem then with setting default locale in routes? Or we should keep it without default locale in url?

Reply:
have you checked what is setting in `default_url_options` method?

Reply:
don't know where to find original of default_url_options is it open source?  Couldn't find working solution till now, maybe that gem could do it? [https://github.com/francesc/rails-translate-routes](url)